lib/store/useTemplateStore.tsx

```diff
@@ -385,7 +388,10 @@ const createTemplateStore = (initProps?: Partial<InitialTemplateStoreProps>) =>
                 state.name = "";
                 state.deliveryOption = undefined;
                 state.formPurpose = "";
-                state.closingDate = null;
+                (state.publishReason = ""),
```

Comment:
Any reason for the ( and ) here?

Reply:
The linter forces it.

Reply:
Hmm not seeing the same or is it something different?

<img width="665" alt="Screenshot 2024-07-08 at 3 18 48 PM" src="https://github.com/cds-snc/platform-forms-client/assets/62242/35f06260-961b-4c6a-b510-dab26e4cb926">

Reply:
Not sure, when I tried to change it to that, the linter just put it back to the brackets before commit.

Reply:
Okay will roll with that and I'll see about making a follow up commit --- once the PR is in --- find it makes it a little harder to read for no reason.